File: Java/ch3/src/ch3/ex3_11.java
package ch3;

public class ex3_11 {
	static int[] makeArr() {
		int temp[] = {1,2,3,4};
		return temp;
	}
	
	public static void main(String[] args) {
		int intArr[] = makeArr();
		
		for(int i = 0 ; i < intArr.length ; i++) {
			System.out.print(intArr[i] + " ");
		}
	}

}

File: Python/python으로 시작하는/ex3_11.py
# 온도에 따른 물 상태 출력
temp = float(input("온도 입력 : "))

print("물의 상태는 ", end="")

if temp <= 0:
    print("얼음")
elif temp > 0 and temp < 100:
    print("액체")
else:
    print("기체")
